frontend/src/components/c-summary-charts.vue

```diff
@@ -6,12 +6,15 @@
     v-bind:style="isChartGroupWidgetMode ? {'marginBottom': 0} : {}"
     )
     .summary-charts__title(
-      v-if="filterGroupSelection !== 'groupByNone' && !isChartWidgetMode",
+      v-if="filterGroupSelection !== 'groupByNone'",
+      v-bind:ref="'summary-charts-title-' + i",
       v-bind:class="{ 'active-background': \
         isSelectedGroup(repo[0].name, repo[0].repoName) && !isChartGroupWidgetMode}"
     )
       .summary-charts__title--index(v-if="!isChartGroupWidgetMode") {{ i+1 }}
-      .summary-charts__title--groupname
+      .summary-charts__title--groupname(
```

Comment:
Can you also try to align the format of the Group by Author titles and single ramp titles with those in the original report

Reply:
Apologies, could you clarify what you mean by this? Currently, the title already seems to be already aligned with the text below when in widget mode (by removing the left padding, as the index is no longer shown). Or do you mean to align it in a different way?

![image](https://user-images.githubusercontent.com/50147457/233606623-c0e7c153-dac6-42b4-8647-dec2b3ba5815.png)

Reply:
Oh I mean the font and styling of the title

Reply:
Ah okay, apologies about that, with the recent refactor in #1979, the new mixins were not imported